### PR TITLE
Blackbox test of Login Screen with use of UiAutomator and Espresso

### PR DESCRIPTION
**Pull request by dbacinski**

### CODE REVIEW

File: app/src/androidTest/groovy/com/macoscope/ketchuplunch/LoginUiAutomatorSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    def "moves to lunch screen when permission is granted and account is chosen by the user"() {
+        given:
+            def uiDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
+        when:
+            loginActivityRule.launchActivity(new Intent(Intent.ACTION_MAIN))
+
+            clickOnTextSilent(uiDevice, "OK")
+            allowCurrentPermission(uiDevice)
+            clickOnText(uiDevice, "darek@macoscope.net")
```
Comment (Tomekkk) [inline app/src/androidTest/groovy/com/macoscope/ketchuplunch/LoginUiAutomatorSpec.groovy]: Tested device needs configured your email account, shouldn't be used some specified test account?

File: app/src/androidTest/groovy/com/macoscope/ketchuplunch/UiAutomatorUtils.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2016 Egor Andreevici
```
Comment (Tomekkk) [inline app/src/androidTest/groovy/com/macoscope/ketchuplunch/UiAutomatorUtils.java]: It is intended?

**Comment by Tomekkk** [comment]
+1

File: app/src/androidTest/groovy/com/macoscope/ketchuplunch/UiAutomatorUtils.java (hunk 2 above)
Comment (dbacinski) [inline app/src/androidTest/groovy/com/macoscope/ketchuplunch/UiAutomatorUtils.java]: Yes. This file was copied from other open source project and refactored.

File: app/src/androidTest/groovy/com/macoscope/ketchuplunch/LoginUiAutomatorSpec.groovy (hunk 1 above)
Comment (dbacinski) [inline app/src/androidTest/groovy/com/macoscope/ketchuplunch/LoginUiAutomatorSpec.groovy]: This test won't be run on CI it only serves as an example how to run Espresso with UiAutomator. 
Currently backend only supports emails form @macoscope.net domain :/

**Comment by pziomacki** [comment]
+2

**Pull request merged**